### PR TITLE
feat: sync issue comment

### PR DESCRIPTION
**Pull request by Mini256**
- [x] feat: sync issue comment
- [x] add test
- [x] add doc

### CODE REVIEW

File: src/events/app/index.ts [hunk 1]
```diff
@@ -241,7 +254,7 @@ async function handleSyncPulls(
 
     // TODO: Optimize the sleep time.
     // In order to avoid frequent access to the API.
-    await sleep(1000);
```
Comment (hi-rustin) [inline src/events/app/index.ts:244]: If we don't add this sleep, will we reach GitHub's limit on how often the GitHub app can access the api?

Comment (Mini256) [reply src/events/app/index.ts:244]: There was no problem with the half-hour test.

Comment (Mini256) [reply src/events/app/index.ts:244]: image
hit rate limit after 50min

Comment (Mini256) [reply src/events/app/index.ts:244]: Maybe probot has handle it for us, It will wait for a while and try again.

Comment (hi-rustin) [reply src/events/app/index.ts:244]: So we can use this mechanism?

Comment (Mini256) [reply src/events/app/index.ts:244]: Yes, but we still need to control the frequency of API access, too fast access will lead to a longer waiting time.

**Review by hi-rustin — APPROVED** [review]

**Pull request merged**